### ISSUE TITLE
Animation plays too fast after upgrade v2 --> v3

### ISSUE THREAD

**Issue by chelovekdrakon** [lottie-react-native/lottie-react-native#577]
### Description

*After upgrade of library from v2.5.11 --> to v3.2.1*
The animation starts playing instantly. 
It plays too fast.

### Repro

v2: https://youtu.be/fPpQOlNxxds
v3: https://youtu.be/VNpqXoKcVPM

Repository: https://github.com/chelovekdrakon/lottie-issue/tree/lottie-animation

### Steps to Reproduce

1. Upgrade library from v2.5.11 --> to v3.2.1

or 

1. Create new react-native app
2. Integrate haul as build manager
3. Fix issue with "use strict" (add "remove-strict-webpack-plugin" to haul.config)
4. Run any animation

**Expected behavior:** After upgrade of the library animation runs the same speed.

**Actual behavior:** After upgrade of the library the animation runs instantly.

### Versions

You can get this information from executing `npm version`.
{
  sharekey: '0.0.1',
  npm: '6.9.0',
  ares: '1.15.0',
  brotli: '1.0.7',
  cldr: '35.1',
  http_parser: '2.8.0',
  icu: '64.2',
  llhttp: '1.1.3',
  modules: '72',
  napi: '4',
  nghttp2: '1.38.0',
  node: '12.3.1',
  openssl: '1.1.1b',
  tz: '2019a',
  unicode: '12.1',
  uv: '1.29.1',
  v8: '7.4.288.27-node.18',
  zlib: '1.2.11'
}

**Comment by chelovekdrakon** [comment]
I don't know, maybe it is connected.
When I tried to use <Image /> to render gif animation --> The same
Animation is too fast.
Here is the repro: https://github.com/chelovekdrakon/lottie-issue/tree/image-gif

**Comment by mchaelml** [comment]
same issue for me, upgraded from 2.5.3

**Comment by kirillpisarev** [comment]
Faced the same issue, did anyone find any solution?

**Comment by chelovekdrakon** [comment]
@kirillpisarev I've tried these versions, but unfortunately, it did not work for me :/

**Comment by chelovekdrakon** [comment]
Interesting that if you add "autoSize" prop to the "LottieView" - it works ok...

**Comment by rayzeller** [comment]
For me, I had to add `flex: 1` to `style` in `LottieView`

**Comment by mayconmesquita** [comment]
Downgrade didn't work. Add `flex: 1` is working

**Comment by tamago3keran** [comment]
It is fixed by v3.4.0 release.
Please try again after upgrading to v3.4.0 ☺️

```js
import React from 'react';
import LottieView from 'lottie-react-native';

export default class App extends React.Component {
  render() {
    return (
      <LottieView
        source={require('./animation.json')}
        autoPlay
        loop={false}
        style={{flex: 1}}
      />
    );
  }
}
```

|v3.3.2|v3.4.0|
|:--:|:--:|
|<img src="https://user-images.githubusercontent.com/24249859/82733315-0085cc80-9d4e-11ea-9df0-7d93e58f8f5e.gif" width=200px>|<img src="https://user-images.githubusercontent.com/24249859/82733695-8a369980-9d50-11ea-87b1-d8790e4d879d.gif" width=200px>|

**Comment by chelovekdrakon** [comment]
@tamago3keran thank you :) That's true, now play speed is right :)
But, this is very strange how it worked before...
Animation JSON is designed for iPhone 8 (screen size), and it was resized very well how would user expect, for any iPhone resolution (X, plus, etc)!
You could see it here: https://youtu.be/fPpQOlNxxds

But now, this magic disappeared :) 
Here is with v3.4.0: https://youtu.be/3nId5KRfBHM

Would be nice at least to understand how it adjusted sized like this before, and what has been changed, that it doesn't work anymore))
